tests/Behat/WebtoolsCookieConsentContext.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_webtools\Behat;
+
+use Behat\Behat\Hook\Scope\AfterScenarioScope;
+use Behat\Behat\Hook\Scope\BeforeScenarioScope;
+use Behat\Gherkin\Node\TableNode;
+use Drupal\DrupalExtension\Context\RawDrupalContext;
+use Drupal\media\Entity\Media;
+
+/**
+ * Behat step definitions related to the oe_webtools_cookie_consent module.
+ */
+class WebtoolsCookieConsentContext extends RawDrupalContext {
+
+  /**
+   * Enables the Media module.
+   *
+   * @param \Behat\Behat\Hook\Scope\BeforeScenarioScope $scope
+   *   The scope.
+   *
+   * @beforeScenario @remote-video
+   */
+  public function enableTestModule(BeforeScenarioScope $scope): void {
+    \Drupal::service('module_installer')->install(['oe_media']);
+  }
+
+  /**
+   * Disables the Media module.
+   *
+   * @param \Behat\Behat\Hook\Scope\AfterScenarioScope $scope
+   *   The scope.
+   *
+   * @afterScenario @remote-video
+   */
+  public function disableTestModule(AfterScenarioScope $scope): void {
```

Comment:
Not a test module. Also, please ensure that behat cleans any created config. If already does, disregard my comment :)

Reply:
Currently oe_media on install create import configs from config/install folder. But after uninstall didn't remove this config because in almost all configs we don't have dependencies from "owner" (oe_media) of this configs. This is how work many other components which provide configs (for instance oe_content, oe_paragraphs etc.). As I understand this is initial approach/decision for config management across all our components.

Reply:
During direct communication with @upchuk, we have decided to left this for now.